spec/yard_sorbet/struct_handler_spec.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require 'yard'
+
+RSpec.describe YARDSorbet::StructHandler do
+  before do
+    YARD::Registry.clear
+    path = File.join(
+      File.expand_path('../data', __dir__),
+      'struct_handler.rb.txt'
+    )
+    YARD::Parser::SourceParser.parse(path)
+  end
+
+  describe 'constructor' do
+    it 'has the appropriate parameters' do
+      node = YARD::Registry.at('PersonStruct#initialize')
+      expect(node.parameters).to eq([['name:', nil], ['age:', nil], ['optional:', 'nil'], ['mystery:', nil]])
```

Comment:
`name` and `age` are required props, and `nil` is not a valid value

Reply:
`nil` means that it is a required value, and for whatever reason, using a literal `'nil'` is how YARD treats a default value here (tested with another method where I did this manually).